src/osd/OSD.cc

```diff
@@ -6457,6 +6491,10 @@ void OSD::do_command(Connection *con, ceph_tid_t tid, vector<string>& cmd, buffe
     ss << "compacted omap in " << duration << " seconds";
   }
 
+  else if (prefix == "smart") {
+    probe_smart(ss);
```

Comment:
ds instead of ss (for these calls ss goes to stderr and ds goes to stdout)

Reply:
thanks, done.